slangpy/tests/slangpy_tests/test_scalar_return.py

```
@@ -0,0 +1,184 @@
+    The return should be a plain Python scalar (via ValueRef), not a
+    torch.Tensor. Creating a 0D torch.Tensor result would crash the Slang
+    compiler on non-CUDA backends.
+    """
+    device = helpers.get_torch_device(device_type)
+    module = helpers.create_module(device, SCALAR_RETURN_SOURCE)
+
+    data = torch.tensor([10.0, 20.0, 30.0], dtype=torch.float32, device="cuda")
+    result = module.read_element(data=data, idx=1.0)
+
+    assert not isinstance(result, torch.Tensor), (
+        f"Expected a scalar return, got torch.Tensor. "
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)